java-backend/src/main/java/org/kframework/backend/java/builtins/BuiltinIOOperations.java

```diff
@@ -114,20 +116,24 @@ public static Term parseInModule(StringToken input, StringToken startSymbol, Str
 
     public static Term system(StringToken term, TermContext termContext) {
         Map<String, String> environment = new HashMap<>();
-        String[] args = term.stringValue().split("\001", -1);
-        //for (String c : args) { System.out.println(c); }
+        String[] args = Commandline.translateCommandline(term.stringValue());
```

Comment:
@dwightguth I use the apache library for this split. A simple split by whitespace can be wrong. Any workarounds so I can avoid including the whole library?

Reply:
just invoke the command via `sh -c`

Reply:
You probably don't want to support shell expressions though, in which case you should create a separate function, `#exec`.

Reply:
In POSIX the `exec` family of functions replaces the current process, so I don't like the idea of reusing that name.

Python calls this functionality `subprocess.call()`
Java: `Runtime.exec()`
Rust: `std::process::Command`'s `spawn`

I think I like `#spawn`

Reply:
That's fair, it's more like a fork-and-exec. Spawn works for me since there doesn't seem to be an immediate posix analogue.